community/bolt/neo4j-home/ssl/snakeoil.cert

```diff
@@ -0,0 +1,10 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
I know these shouldn't be here but this is a revert I'll reremove when rereverting...